core/src/main/java/org/apache/calcite/sql/SqlKind.java

```diff
@@ -1215,6 +1218,7 @@ public enum SqlKind {
                   FILTER, WITHIN_GROUP, IGNORE_NULLS, RESPECT_NULLS, SEPARATOR,
                   DESCENDING, CUBE, ROLLUP, GROUPING_SETS, EXTEND, LATERAL,
                   SELECT, JOIN, OTHER_FUNCTION, POSITION, CAST, TRIM, FLOOR, CEIL,
+                  DATE_ADD, TIMESTAMP_ADD, TIMESTAMP_DIFF, EXTRACT, INTERVAL,
```

Comment:
EXTRACT, INTERVAL and maybe others are now duplicated.

Reply:
This got messed up during rebase, sorry about that.